homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -0,0 +1,73 @@
+"""Suppoort for Amcrest IP camera binary sensors."""
+import asyncio
+from datetime import timedelta
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.const import CONF_NAME, CONF_BINARY_SENSORS
+from . import DATA_AMCREST, BINARY_SENSORS
+
+DEPENDENCIES = ['amcrest']
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=5)
+
+
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
```

Comment:
use `async def` instead of `@asyncio.coroutine`

Reply:
Good catch. Will do! (It's been so long since I originally wrote this I forgot about that.)